### ISSUE TITLE
Temperature not pulling from PPBAdvance when using ZWO EAF

### ISSUE THREAD

**Issue by swatts-godaddy** [jlecomte/ascom-virtual-focuser#4]
I have downloaded this Virtual Focuser ASCOM driver to use with my ZWO EAF and PPBAdvance for temperature readings. When setting this up in NINA, I see the description is set correctly, but the temperature reading of the focuser does not match the temperature from the PPBAdvance and still seems to be pulling from the EAF.

![image](https://github.com/jlecomte/ascom-virtual-focuser/assets/89647779/03301d25-ccd8-4d4c-a080-282bf49b05a0)

**Comment by swatts-godaddy** [comment]
I believe the issue is here, https://github.com/jlecomte/ascom-virtual-focuser/blob/master/ASCOM_Driver/FocuserDriver.cs#L169 its falling back to the `focuser` temperature even if it successfully read the temperature from the `observingConditionsDevice`.  Therefore will work when the focuser doesn't have a temperature reading.

Changing this line to the following should work
```
if (!observingConditionsDeviceHasTemperatureSupport && focuser != null)
```

**Comment by jlecomte** [comment]
Right, that is what I was looking at. Should be a relatively simple fix. Let me do that now.

**Comment by jlecomte** [comment]
Fixed in https://github.com/jlecomte/ascom-virtual-focuser/commit/16afa07a8a42666b9fc3ddbade760d5d3ca82c60 and released `v1.2.1`. Please test and report back. Thanks!

**Comment by swatts-godaddy** [comment]
Tested and confirmed its working now. Thanks for the quick fix.
